src/gui/screen_print_preview.cpp

```diff
@@ -313,13 +313,17 @@ static int screen_print_preview_event(screen_t *screen, window_t *window,
     }
 
     if (!suppress_draw && fs_did_filament_runout()) {
+        rect_ui16_t msg_box_size = gui_defaults.scr_body_no_foot_sz;
+        msg_box_size.y = 0;
+        msg_box_size.h += gui_defaults.header_sz.h;
         suppress_draw = true;
-        Sound_Play(eSOUND_TYPE_StandardAlert);
+
+        Sound_Play(eSOUND_TYPE_SingleBeep);
         const char *btns[3] = { N_("YES"), N_("NO"), N_("IGNORE") };
-        switch (gui_msgbox_ex(string_view_utf8::MakeNULLSTR(),
+        switch (gui_msgbox_ex(string_view_utf8::MakeRAM((const uint8_t *)gcode_file_name),
```

Comment:
document, that using MakeRAM in this case is safe, because gcode_file_name lives long enough
... I'm still thinking about renaming MakeRAM to something obscure like: `IKnowTheContainedPointerWillLiveLongEnoughWhenWrappingItIntoThisStringViewObject()` :)

Reply:
We talked about this case with gcode_file_name and even though I love that new name, I don't think is short enough ... MakeRAM is a little bit confusing, I suggest "MakeText" or "WrapText" ?

Reply:
yes, it should be something like FromRAM or similar ... still awaiting better suggestions ;) .